src/utilities/dates.ts

```diff
@@ -125,29 +98,29 @@ export function getNewRange(range: Range | undefined, selected: Date): Range {
   return {start: selected, end: selected};
 }
 
-export function getNextDisplayMonth(month: Month): Month {
-  if (Month[month] === Month[11]) {
+export function getNextDisplayMonth(month: number): number {
+  if (month === 11) {
```

Comment:
"what about numbers bigger than 11" I hear you cry. Well it's always been possible to pass numbers bigger than 11 into this function and it's not been a problem yet :/